### DISCUSSION TITLE
記事の下の方を見ていると「編集する」ボタンが押せない【限定共有記事】

### DISCUSSION THREAD

**Discussion by stk2k** [increments/qiita-discussions#217] [要望・アイデア・不具合]
自分が投稿した記事を修正したいとします。

１）記事の一番先頭が見えている状態
→…ボタンを押す→「編集する」メニューが表示される→編集開始

２）記事の中ほどまでスクロールした状態
→…ボタンを押す→「編集する」メニューがブラウザの上部からはみ出てしまい押せない
→上までスクロールする→１）へ

となり、地味にストレスを感じます・・・

そもそも自分の記事は投稿後、

・誤字などを見つけて修正したい→編集ボタンを探す
・ヤバい表現があって非公開に戻したい→非公開ボタンを探す
・投稿したが恥ずかしくなったので消したい→削除ボタンを探す

といういずれかの動機だと思いますので、「…」というアクションを１つ挟むのではなくダイレクトに上記３つのボタンが欲しいところではあります。。。

あるいは、「…」はあってもいいのですがせめてフローティングにしていつでも押せるようにしていただければ編集がはかどるのではないでしょうか。

![449441ec50705c4c68a26a0e785be4a1](https://user-images.githubusercontent.com/985640/167414414-f09dc33d-14c3-4ba4-9092-2dec583952bf.png)

参考：「編集する」リンクが見切れている

**Comment by stk2k** [comment]
本Discussionとは逸れてしまいますが

・全体に公開する
・編集履歴
・Markdownで本文を見る
・削除

と、メニュー名の統一感がなく、脳が一瞬思考停止します。
体言止めなら体言止め、動詞形なら動詞形で統一されてはいかがでしょう。

≪体言止め≫
・全体（に）公開
・編集履歴
・Markdown確認
・削除

≪動詞形≫
・全体に公開する
・編集履歴を見る
・Markdownで本文を見る
・記事を削除する

**Comment by tanaka-asami415** [comment]
@stk2k 
ご意見ありがとうございます🙏

我々でも確認してみたところ、通常の投稿では再現できなかったのですが、
こちらは限定共有記事のみで起きている現象でしょうか？
限定共有記事はこちらでも確認できたので、修正いたします！

> 体言止めなら体言止め、動詞形なら動詞形で統一されてはいかがでしょう。

こちらもありがとうございます。今後の改修の際に、検討させていただきます🙏

**Comment by stk2k** [comment]
@tanaka-asami415 

限定記事だけの現象ですね・・・失礼いたしました。
（本Discussionのタイトルに追記しました）

通常の投稿の場合、LGTMとストックボタンがある分、メニューが上に行かないので結果的に「編集」ボタンが隠れない状況のようです。
限定共有記事について、修正をお待ちしております。

>>体言止めなら体言止め、動詞形なら動詞形で統一されてはいかがでしょう。

>こちらもありがとうございます。今後の改修の際に、検討させていただきます🙏

重要度は低いですが、お願いいたします。

**Comment by kyntk** [comment]
@stk2k 大変遅くなりましたが、限定共有記事でボタンが押せない不具合を修正しました！

**Comment by stk2k** [comment]
@kyntk 修正ありがとうございます。問題が修正されていることを確認しました。使い勝手がさらにUPしました！
クローズの方よろしくお願いします。

**Comment by wataru86** [comment]
@stk2k
大変お待たせしました 🙇 
メニュー名を動詞形に変更する対応も完了しました :sparkles:

- 編集履歴 -> 編集履歴を見る
- 削除 -> 削除する

**Reply by stk2k** [reply]
@wataru86
修正内容確認いたしました。とても見やすくなりました！ありがとうございました。
